### DISCUSSION TITLE
Playlist extractor : How to get videos IDs list ?

### DISCUSSION THREAD

**Discussion by Miala-python** [TeamNewPipe/NewPipe#10879] [Questions]
Hello, I'm interested about how the YouTube playlist extractor works for another project, but I don't know Java. 
I tried to read the documentation but I didn't understand what YouTube URLs are used. 😅
Can you explain to me how we can get the entire playlist (I'm only interested in the list of video IDs) ? 
Thank you !

**Comment by opusforlife2** [comment]
Ask this in the extractor repo?
